equipment_handler/source/housekeeping_charon.c

```diff
@@ -19,14 +19,18 @@
 
 #include "housekeeping_charon.h"
 
-int Charon_getHK(charon_housekeeping * hk){
+GPS_RETURNSTATE Charon_getHK(charon_housekeeping * hk){
     // Read temperature sensors
     if(readAllTemps(&hk->temparray[0])){
```

Comment:
What does this do exactly?

Reply:
It's a function for the housekeeping task to call that collects the temp data from I2C and the fimware CRC from the skytraq

Reply:
What I mean is what is the is statement checking for?